src/utilities/formatToProposal.ts

```diff
@@ -51,22 +87,13 @@ const formatToProposal = ({
     descriptionObj = { version: 'v1' as const, title: plainTitle, description: descriptionText };
   }
 
-  if (!endDate) {
```

Comment:
Are we guaranteed to have an endDate? I took this from the original implementation that was calculate the end date based on the endBlock if it was missing

Reply:
Nope, that's a mistake on my end! Good catch